### ISSUE TITLE
Performance measures for your Data Science/Machine Learning application 1.

### ISSUE THREAD

**Issue by munubi254** [section-engineering-education/engineering-education#3035]
### Proposed title of the article
Performance measures for your Data Science/Machine Learning application 1.

### Introduction paragraph (2-3 paragraphs):
While working on our models, we would at one point need to check their performance. This would include checking on the errors, the accuracy, biases, etc. We would do that in this article using a classification problem. Since we would look at them in depth, I will look at checking the errors using Cross-validation scores and the Confusion matrix in this part.

We will be using Python and Scikit-Learn.

### Key takeaways: 
The takeaways are checking the performance using:

1. Cross-validation scores.
2. Confusion matrix.

Moreover, we will look at using a Stochastic Gradient Classifier on a dataset.

### References: 
Please list links to any published content/research that you intend to use to support/guide this article.

BOOK: Hands-On Machine Learning with Scikit-Learn & TensorFlow CONCEPTS, TOOLS, AND TECHNIQUES TO BUILD INTELLIGENT SYSTEMS by Aurélien Géron.

**Comment by hectorkambow** [comment]
@munubi254 
Good afternoon and thank you for submitting your topic suggestion.
Your topic form has been entered into our queue and should be reviewed (for approval) as soon as a content moderator is finished reviewing the ones in the queue before it.

**Comment by hectorkambow** [comment]
@munubi254 
Sounds like an interesting topic - let me ping our resident ML expert to see how we can draw any more substance from this particular topic @lalith1403

**Comment by lalith1403** [comment]
@munubi254 Thanks for the topic suggestion. Can you clarify the purpose of the article. We tend to publish articles that are programmer-centric. Confusion matrix and cross-validation are small topics and well documented. We would love to hear from you about the unique aspect that you would bring with this article. 

Probably, a project that uses these concepts to assess the performance and upgrade it by using these metrics as a navigating guide. I hope that makes sense.

Cc: @hectorkambow

**Comment by munubi254** [comment]
Yes @lalith1403. I'll be doing that using a classification task based on
the MNIST dataset. Thereafter, I will do the performance measures using the
metrics.

**Comment by lalith1403** [comment]
MNIST dataset is a toy dataset and well documented throughout the net. We are looking to publish articles that are programmer-centric and unique. Thanks for the topic suggestion. We encourage you to submit a new topic form with a new idea. Thank you.

I will be closing this issue for now. Look forward to seeing your new idea. Ping me once done.
